src/assets/javascripts/components/Material/Search/Result.jsx

```diff
@@ -23,6 +23,9 @@
 import escape from "escape-string-regexp"
 import lunr from "lunr"
 
+window.lunr = lunr
```

Comment:
This breaks with all the nice encapsulation and exposes `lunr` to the window. Pleae find another way to do this that doesn't involve putting objects into the global object. There should at any time only be a single entrypoint and that is `app` in `application.js`